README.md

```diff
@@ -31,10 +32,33 @@ Any IDE will work just fine. VSCode, Visual Studio Community, CLion, etc.
 Running the app:
 
 cd to the the app folder and run
-- npm start
+- `npm start`
 
 You can run the app either in Android emulator, IOS simulator (only if you have a Mac), on the web browser, or with the Expo Client App on your phone. 
 
-To run the app in Android emulator you have to install Adriod Studio. For more detial
-watch the video suggested above. 
-
+To run the app in Android emulator, you have to install Andriod Studio. For more detial watch the video suggested above. 
+
+
+Development and Code dependencies:
+
+We make use of many open-source libraries from the react-native communities. Although,
+all the dependencies should be included in the project; if some are missing, check the 
+package.json file and make sure that it have all the following dependencies:
```

Comment:
We do not need to duplicate the contents of `package.json` in the README. We can delete this section.

Also, does the README include the command to install all of the dependencies from `package.json`? If not, we should include that command.